integration/js/utils/SdkBaseTest.js

```diff
@@ -15,6 +15,9 @@ class SdkBaseTest extends KiteBaseTest {
     if (testName === 'ContentShareOnlyAllowTwoTest') {
       this.url = this.getTransformedURL(this.url, 'max-content-share', 'true');
     }
+    if (['ContentShare'].includes(testName)) {
```

Comment:
Should this be the other way?
`testName.includes('ContentShare')`

Reply:
Our testName can be 'ContentShareScreenCapture', 'ContentShareOnlyAllowTwoTest' or 'MeetingLeaveContentShareTest', so this check would not pass right?